autoprecompiles/src/openvm.rs

```diff
@@ -24,3 +24,72 @@ pub fn default_openvm_bus_map() -> BusMap {
     .collect();
     BusMap::new(bus_ids)
 }
+
+// Rv32BaseAluChip
```

Comment:
This hard-codes some OpenVM opcodes inside the `autoprecompiles` crate. I think the entire file can be moved to the `openvm` crate (including `default_openvm_bus_map`, which existed here before this PR)?

Reply:
Done here https://github.com/powdr-labs/powdr/pull/2889